audibot_description/urdf/audibot.urdf.xacro

```diff
@@ -146,6 +146,19 @@
     <child link="base_link"/>
   </joint>
 
+  <link name="front_axle">
+    <inertial>
+      <mass value="0.1"/>
+      <inertia ixx="0.0001" ixy="0.0001" ixz="0.0001" iyy="0.0001" iyz="0.0001" izz="0.0001" />
+    </inertial>
+  </link>
+
+  <joint name="front_axle_joint" type="fixed">
+    <origin xyz="${2*half_wheelbase} 0 0" rpy="0 0 0"/>
+    <parent link="base_footprint"/>
+    <child link="front_axle"/>
+  </joint>
+
```

Comment:
I don't agree with adding this link and joint to the base URDF model because it does not serve any purpose for the behavior of the simulated vehicle.

Instead, I would recommend this:

1. Create a new URDF model in a separate ROS package
2. Include this base URDF model in the new one
4. Add custom links, sensors, etc. specific to your simulation goals
5. Write a modified `audibot_robot.launch` to load your new custom URDF model into Gazebo

Such a custom URDF file would look something like this:
```
<robot name="custom_vehicle_sim" xmlns:xacro="http://www.ros.org/wiki/xacro" >
  <xacro:include filename="$(find audibot_description)/urdf/audibot.urdf.xacro" />

  <link name="front_axle">
    <inertial>
      <mass value="0.1"/>
      <inertia ixx="0.0001" ixy="0.0001" ixz="0.0001" iyy="0.0001" iyz="0.0001" izz="0.0001" />
    </inertial>
  </link>

  <joint name="front_axle_joint" type="fixed">
    <origin xyz="2.652 0 0" rpy="0 0 0"/>
    <parent link="base_footprint"/>
    <child link="front_axle"/>
  </joint>

  <gazebo reference="front_axle_joint" >
    <preserveFixedJoint>true</preserveFixedJoint>
  </gazebo>
</robot>
```